charts/timed/templates/cronjob-backend.yaml

```
@@ -0,0 +1,33 @@
+{{- range $key, $val := .Values.backend.cronjobs }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "timed.fullname" . }}-{{ $key }}
```

You need to use the root context inside of `range`. I think something like this works:
```suggestion
  name: {{ include "timed.fullname" $ }}-{{ $key }}
```